tests/src/test/scala/system/basic/WskUnicodeTests.scala

```diff
@@ -64,36 +64,39 @@ class WskUnicodeTests extends TestHelpers with WskTestHelpers with JsHelpers wit
     val bin = new File(TestUtils.getTestActionFilename(s"$prefix.bin"))
     if (txt.exists) Some(txt.toString)
     else if (bin.exists) Some(bin.toString)
-    else throw new IllegalStateException(s"did not find text or binary action for kind $kind")
+    else {
+      println(s"WARNING: did not find text or binary action for kind $kind, skipping it")
+      None
+    }
   }
 
-  actionKinds.foreach { k =>
-    val actionKind = k.kind
-
-    s"$actionKind action" should "Ensure that UTF-8 in supported in source files, input params, logs, and output results" in withAssetCleaner(
-      wskprops) { (wp, assetHelper) =>
-      val name = s"unicodeGalore.${actionKind.replace(":", "")}"
-
-      assetHelper.withCleaner(wsk.action, name) { (action, _) =>
-        action.create(
-          name,
-          getFileLocation(actionKind),
-          main = main(actionKind),
-          kind = Some(actionKind),
-          timeout = Some(activationMaxDuration))
+  // tolerate missing files rather than throw an exception
+  actionKinds.map(k => (k.kind, getFileLocation(k.kind))).filter(_._2.isDefined).foreach {
+    case (actionKind: String, file: Option[String]) =>
```

Comment:
You could simplify this by doing:

Reply:
let's do that cleanup in a subsequent PR.  would like to merge this one as soon as we get a clear from travis.